Cargo.toml

```diff
@@ -9,6 +9,21 @@ edition = "2018"
 name = "polymesh"
 path = "src/main.rs"
 
+[workspace]
```

Comment:
Now that we have the workspace, we can also remove the `Cargo.lock` file in each member.